app/javascript/react/components/MapButtons/MapButtons.tsx

```diff
@@ -106,21 +114,19 @@ const MapButtons = () => {
             <MapButton
               title={t("navbar.copyLink")}
               image={copyLinkIcon}
-              onClick={() => {}}
+              onClick={() => handleCopyLinkClick()}
               alt={t("navbar.altCopyLink")}
-              isActive={activeButtons.includes(ButtonTypes.COPY_LINK)}
+              isActive={activeCopyLinkButton}
```

Comment:
I would unify the naming and have `activeFiltersButton` or rename this to `isCopy LinkButtonActive`